go-controller/pkg/ovn/policy.go

```diff
@@ -103,6 +103,10 @@ func hashedPortGroup(s string) string {
 	return util.HashForOVN(s)
 }
 
+func trimNameRFC1123(name string) string {
+	return fmt.Sprintf("%.63s", name)
```

Comment:
you may want to introduce a randomized suffix (with additional trimming) to avoid collapsing `really-long...-a` and `really-long-...-b` into conflicting names.

Reply:
yes but this is already an existing problem prior to this PR, @kyrtapz and I were also thinking the same that we are going to end up with conflicting names because of the trimming but that's ok for ACLs, because the name isn't unique; its totally ok for multiple ACLs to have the same name.

Will do a more long term solution in the future where we'd need to clean up names for all ACLs. -> need to bring it up with the team on if we need to trim before or after adding the `egressDefaultDeny`.

//slightly unrelated, I realised the update stale acts is now going to be called over these trimmed out acls since they won't have the `egressDefaultDeny` tag, but good thing is they will get replaced by the same trimmed name.